video_frame_metadata_registry.src.html

```diff
@@ -61,6 +61,18 @@
     <td>segments</td>
     <td>[Human face segmentation](https://w3c.github.io/mediacapture-extensions/#human-face-segmentation)</td>
   </tr>
+  <tr>
+    <td>captureTime</td>
+    <td>[Capture time](https://wicg.github.io/video-rvfc/#dom-videoframecallbackmetadata-capturetime)</td>
+  </tr>
+  <tr>
+    <td>receiveTime</td>
+    <td>[Receive time](https://wicg.github.io/video-rvfc/#dom-videoframecallbackmetadata-receivetime)</td>
```

Comment:
Comment: can we allow this to be present in a WebCodecs application as well (set by the receiver)?